pkg/pkg.bzl

```diff
@@ -503,68 +504,171 @@ def pkg_tar(name, **kwargs):
 pkg_deb_impl = rule(
     implementation = _pkg_deb_impl,
     attrs = {
```

Comment:
Maybe add a comment with `# @unsorted-dict-items` to prevent aggressive `buildifier` runs from resorting this?

Reply:
Done